src/mscorlib/shared/System/Collections/Generic/Dictionary.cs

```diff
@@ -759,6 +759,24 @@ IEnumerator IEnumerable.GetEnumerator()
             return new Enumerator(this, Enumerator.KeyValuePair);
         }
 
+        /// <summary>
+        /// Ensures that the capacity of this dictionary is at least the given minimum value
+        /// </summary>
+        /// <param name="capacity"></param>
```

Comment:
Please don't include empty doc comments. If they're empty, remove them (seems we don't consistently use them) . 

It's just noise.